net/shorewall/patches/020-init_sh.patch

```diff
@@ -0,0 +1,97 @@
+Index: shorewall-5.1.4.1/init.openwrt.sh
+===================================================================
+--- /dev/null	1970-01-01 00:00:00.000000000 +0000
++++ shorewall-5.1.4.1/init.openwrt.sh	2017-06-06 22:42:37.738055621 +0200
+@@ -0,0 +1,92 @@
++#!/bin/sh /etc/rc.common
++#
++#     The Shoreline Firewall (Shorewall) Packet Filtering Firewall - V4.5
++#
++#     (c) 1999,2000,2001,2002,2003,2004,2005,2006,2007,2012,2014 - Tom Eastep (teastep@shorewall.net)
++#     (c) 2015 - Matt Darfeuille - (matdarf@gmail.com)
++#
++#	On most distributions, this file should be called /etc/init.d/shorewall.
++#
++#	Complete documentation is available at http://shorewall.net
++#
++#       This program is part of Shorewall.
++#
++#	This program is free software; you can redistribute it and/or modify
++#	it under the terms of the GNU General Public License as published by the
++#       Free Software Foundation, either version 2 of the license or, at your
++#       option, any later version.
++#
++#	This program is distributed in the hope that it will be useful,
++#	but WITHOUT ANY WARRANTY; without even the implied warranty of
++#	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
++#	GNU General Public License for more details.
++#
++#	You should have received a copy of the GNU General Public License
++#	along with this program; if not, see <http://www.gnu.org/licenses/>.
++#
++#	If an error occurs while starting or restarting the firewall, the
++#	firewall is automatically stopped.
++#
++#	Commands are:
++#
++#	   shorewall start		  Starts the firewall
++#	   shorewall restart		  Restarts the firewall
++#	   shorewall reload		  Reload the firewall
++#	   shorewall stop		  Stops the firewall
++#	   shorewall status		  Displays firewall status
++#
++
++# description: Packet filtering firewall
++
++# Openwrt related
++# Start and stop runlevel variable
++START=50
++STOP=89
++# Displays the status command
++EXTRA_COMMANDS="status"
++EXTRA_HELP=" status Displays firewall status"
++
++################################################################################
++# Get startup options (override default)
++################################################################################
++OPTIONS=
++
++#
++# The installer may alter this
++#
```

Comment:
please re add
. /usr/share/shorewall/shorewallrc

Reply:
Corrected.

Reply:
Should this be a patch or just a file copied out of `files/etc/` or wherever?  Also, why do we need this in addition to the other startup scripts?  Is this for the non-procd cases?